ansible/roles_ocp_workloads/ocp4_workload_example_test/meta/main.yml

```diff
@@ -0,0 +1,53 @@
+galaxy_info:
```

Comment:
Do we need this big old file? The other workloads have a much more compact file here.

Reply:
deleted file